src/modules/form/components/client/emails/EmailInput.tsx

```diff
@@ -1,16 +1,16 @@
 import { Grid, Stack } from '@mui/material';
 
 import { getRequiredLabel } from '../../RequiredLabel';
-import { PhoneInputType } from '../types';
+import { EmailInputType } from '../types';
 
 import TextInput from '@/components/elements/input/TextInput';
 
-const PhoneInput = ({
+const EmailInput = ({
```

Comment:
See my comment below for more backstory, but when I was searching the codebase for usages of `PhoneInput`, I noticed this and thought it was probably a typo from an incomplete copy/paste?

Reply:
yep  thanks for fixing!